heat/core/dndarray.py

```diff
@@ -705,6 +704,12 @@ def create_partition_interface(self):
             "partitions": partitions,
             "locals": [tuple(lcls)],
         }
+
+        def _partition_getter(key):
+            return partition_dict["partitions"][key]["data"]
```

Comment:
The idea was to accept whatever is located in `partition_dict["partitions"][key]["data"]`.
A user would use it like this `pdict['get'](pdict['partitions"][key]["data"]`.

Reply:
here, it's really `lambda x: x`

Reply:
ah! i was thinking one level of abstraction more than that.

Reply:
now this is what the snippet would look like:
```python
    x = ht.arange(8* 3* 2).reshape((8, 3, 2)).resplit(0)
    print(x.__partitioned__['get'](x.__partitioned__['partitions'][(0,0,0)]['data']))
```